src/node/node_class.hpp

```diff
@@ -1086,6 +1101,8 @@ inline std::vector<Napi::PropertyDescriptor> ObjectWrap<ClassType>::create_napi_
 template<typename ClassType>
 Napi::Object ObjectWrap<ClassType>::create_instance_by_schema(Napi::Env env, Napi::Function& constructor, const realm::ObjectSchema& schema, Internal* internal) {
 	Napi::EscapableHandleScope scope(env);
+	auto& s_schemaObjectTypes = get_schemaObjectTypes();
+	ClassType& s_class = get_class();
```

Comment:
Not a big deal, but it seems odd to sometimes use `auto` for one of these and the actual type for the other. Maybe just always use auto?

Reply:
will fix